src/adapter/src/notice.rs

```diff
@@ -163,6 +166,12 @@ impl fmt::Display for AdapterNotice {
                     IsolationLevel::Serializable.as_str()
                 )
             }
+            AdapterNotice::DroppedSubscribe { dropped_name } => {
+                write!(
+                    f,
+                "subscribe has been dropped because underlying relation {dropped_name} was dropped"
```

Comment:
has been terminated?

Reply:
Yeah, I like that wording much better. Just updated it, thanks for the suggestion!